crmlogin/Contact.php

```diff
@@ -0,0 +1,142 @@
+<?php
+
+/**
+Methods used to edit contacts. Used in actualContactEdit.php
+*/
+	class Contact{
+		
+
+		private $userInfo = array('firstname' => " ", 'lastname' => " ", 'email' => " ");
+		private $user; 
+		private $currentEmail;
+
+/**
+Instantiates the object by email
+*/
+		public static function GetByEmail($email){ 
+			include 'connection.php';
+			session_start();
+			global $user;
+			$user = $_SESSION['user'];
+			if(!isset($user)){
+				header("Location: index.html");
+			}
+
+			global $userInfo;
+			global $currentEmail;
+			$sql = "SELECT * FROM `contactlist` WHERE `email`='$email' AND `user`='$user'";
+			$result = $conn->query($sql);
+			$info = $result->fetch_assoc();
+
+			$userInfo['firstname'] = $info['firstname'];
+			$userInfo['lastname'] = $info['lastname'];
+			$userInfo['email'] = $info['email'];
+			$currentEmail = $info['email'];
+			// echo $userInfo['email'];
+			// echo $currentEmail;
+			//echo $userInfo['firstname'];
+			
+		}
+
+/**
+Instantiates the object by contactId
+*/
+		public static function GetById($id){
+			include 'connection.php';
+			session_start();
+			global $user;
+			$user = $_SESSION['user'];
+			if(!isset($user)){
+				header("Location: index.html");
+			}
+
+			global $userInfo;
+			global $currentEmail;
+			$sql = "SELECT * FROM `contactlist` WHERE `contactId`='$id' AND `user`='$user'";
+			$result = $conn->query($sql);
+			$info = $result->fetch_assoc();
+
+			$userInfo['firstname'] = $info['firstname'];
+			$userInfo['lastname'] = $info['lastname'];
+			$userInfo['email'] = $info['email'];
+			$currentEmail = $info['email'];
+			// echo $userInfo['email'];
+			// echo $currentEmail;
+			//echo $userInfo['firstname'];
+			
+		}
+/**
+Sets the value of a column you want to edit.
+$set can be firstname, lastname, or email
+$value is the new value you want to store
+*/
+		public function setValue($set, $value){
+			include 'connection.php';
+
+			global $userInfo;
+			$userInfo[$set] = $value;
+			
+
+		}
+/**
+Gets the column from the database and returns it.
+*/
+		public function getValue($value){
+			global $userInfo;
```

Comment:
no globals, use class variables... private userInfo declared at the top... and then pull from that